### ISSUE TITLE
Add no-JavaScript/old browser fallback

### ISSUE THREAD

**Issue by twm** [twm/yarrharr#149]
Yarrharr should do something reasonable when it can't load properly:

* #145 introduced [CSS grid](https://caniuse.com/#feat=css-grid) and [CSS variables](https://caniuse.com/#feat=css-variables). Flexbox is also in use. Old browsers won't interpret Yarrharr's CSS properly.
* #126 (shipping ES2015) means that old browsers will get a syntax error on the JS, meaning nothing displays at all!
* Loading JS may fail in general, because networks. It would be nice to do something sensible.

Basically there are two conditions:

1. Yarrharr failed to load for whatever reason. Tell the user to reload the page.
2. Yarrharr cannot display in this browser. Sorry?

The first point above can easily be tested in IE 11.

**Comment by twm** [comment]
This was addressed by switching to an MPA in #845.
